site_scons/components/__init__.py

```diff
@@ -134,6 +134,7 @@ def define_mercury(reqs):
                 libs=['fabric'],
                 config_cb=ofi_config,
                 headers=['rdma/fabric.h'],
+                pkgconfig='libfabric',
```

Comment:
Does the pkgconfig still come from source? I.e. from https://github.com/ofiwg/libfabric?

Reply:
yes

Reply:
https://github.com/ofiwg/libfabric/blob/main/libfabric.pc.in

Reply:
fwiw I checked that --build-deps=yes with and without USE_INSTALLED=ofi has mercury pick up the right libfabric

Reply:
Thanks :)